includes/payment-methods/class-wc-stripe-express-checkout-helper.php

```
@@ -629,7 +629,52 @@ public function get_product() {
+	}
+
+	/**
+	 * Returns true if the current page is a subscription change payment method page.
+	 *
+	 * @return boolean
+	 */
+	public function is_change_payment_method_page() {
+		return isset( $_GET['change_payment_method'] ) // phpcs:ignore WordPress.Security.NonceVerification
+			&& function_exists( 'wcs_is_subscription' )
+			&& wcs_is_subscription( absint( $_GET['change_payment_method'] ) ); // phpcs:ignore WordPress.Security.NonceVerification
+	}
```

PHPMD (2.15.0)

[error] is_change_payment_method_page accesses the super-global variable $_GET. (undefined)

(Superglobals)